tests/router_test.ts

```diff
@@ -38,50 +38,50 @@ let scenarios = [
   {
     name: 'Sync Get Handler',
     async: false,
-    getRoute: function(name: string) {
+    getRoute: function (name: string) {
```

Comment:
Lots of unrelated changes here (by the looks of them they are prettier@2 related), can you roll them back?

Reply:
Done